rmw/include/rmw/types.h

```diff
@@ -234,6 +241,7 @@ typedef struct RMW_PUBLIC_TYPE rmw_event_t
   void * data;
   /// The event type that occurred.
   rmw_event_type_t event_type;
+  rmw_event_data_type_t event_data_type;
```

Comment:
I think the name `rmw_event_data_type_t` is correct since it references the type of `void * data`
https://github.com/irobot-ros/rmw/pull/19/files#diff-5077cc204abb0905a87b794aeb6a1ead49b94eb039668c7bea4d76664a6f8172L234